pom.xml

```diff
@@ -3,8 +3,6 @@
     <modules>
         <module>javaparser-core</module>
         <module>javaparser-testing</module>
-        <module>javaparser-core-generators</module>
```

Comment:
Well, they are not really modules anymore. They are now stand-alone projects that use the `javaparser-parent` module as their parent. So I guess in intellij for example, you would have to treat them as separate projects.